.github/workflows/maven.yml

```diff
@@ -27,4 +27,20 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-jdk-${{ matrix.java }}-mvn-
       - name: Build it with Maven
-        run: mvn clean verify -B -Pqulice
+        run: |
+          if [ "$RUNNER_OS" == "Linux" ]; then
+             mvn clean verify -B -Pqulice
+          elif [ "$RUNNER_OS" == "Windows" ]; then
+               mvn clean test -B -Pqulice
+          else
+             echo "$RUNNER_OS not supported"
+             exit 1
+          fi
```

Comment:
This should be done on test case level - no need to disable all tests for OS, just use `@DisabledOnOs` annotation for test case: https://junit.org/junit5/docs/5.3.0/api/org/junit/jupiter/api/condition/DisabledOnOs.html